src/components/tracker.js

```diff
@@ -0,0 +1,48 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { useNavigate } from 'react-router-dom';
+import { fetchTacker, setRegions } from '../Redux/tracker/tracker';
+import WorldMap from '../assets/worldmap.png';
+
+const Tacker = () => {
```

Comment:
- [ ] While the code works anyway it would be nice to be consistent with our names. You probably meant to write **Tracker** here. Kindly fix it.
- [ ] Kindly look at the image. Your Tracker component (Homepage) is missing the filtering functionality as stated in the requirements. Kindly add a search or filtering mechanism to your homepage
![sunny-home](https://user-images.githubusercontent.com/36542986/142729026-9f9e9d5d-5fab-456a-a50b-c14dad87614f.jpg)
- [ ] Take a look at the image below. There seems to be an issue with your **alternating-dual color**. Please resolve your color pattern issue to be similar to what we have on the template
![amal-alt-color](https://user-images.githubusercontent.com/36542986/142729160-f550cbfc-9527-4ec9-8fcd-726d80fe2f60.jpg)